pubspec.yaml

```diff
@@ -8,5 +8,10 @@ environment:
   # Tested with 3.0.6, but others should work too
   sdk: '>=3.0.0 <3.1.0'
 
+dependencies:
+  args: ^2.4.2
+  collection: ^1.18.0
+  yaml: ^3.1.2
```

Comment:
I'm talking about that. These packages are only used in add_imports, which is only used as a dev script